cpp/CMakeLists.txt

```diff
@@ -84,7 +84,9 @@ if(WIN32)
     endif()
 else()
     if(${ARCTICDB_USING_CONDA})
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra")
+        # Required to be able to include headers from glog since glog 0.7
+        # See: https://github.com/google/glog/pull/1030
+        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra -DGLOG_USE_GLOG_EXPORT")
```

Comment:
You don't need to change this but setting the CMAKE_CXX_FLAGS is an outdated approach. Unfortunately most of our code is setup like this :(

The correct thing to do is to use `target_compile_definitions`